kernel/Dockerfile

```diff
@@ -98,8 +98,16 @@ RUN make defconfig && \
     cp System.map /out && \
     ([ -n "${DEBUG}" ] && cp vmlinux /out || true)
 
+# WireGuard
+RUN TAG="0.0.$(curl -sSL https://git.zx2c4.com/WireGuard/info/refs/ | sed -n 's/.*refs\/tags\/0\.0\.\([0-9]\+\)$/\1/p' | sort -nr | head -n 1)" && \
+    git clone https://git.zx2c4.com/WireGuard/ && \
+    git -C WireGuard verify-tag -v "$TAG" && \
+    git -C WireGuard checkout -b docker-build "$TAG" && \
+    make -j "$(getconf _NPROCESSORS_ONLN)" M="$PWD/WireGuard/src" modules
```

Comment:
I would also suggest to explicitly checkout wireguard in, say `/WireGuard` and then replace the use of `$PWD` with the fixed path